ipc/zeromq.py

```diff
@@ -0,0 +1,50 @@
+#
+#  Copyright (C) 2017, Collabora Ltd.
+#
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this file,
+#  You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+import os
+import zmq
+import os.path
+
+IPC_SOCKET_PATH = os.path.join(os.environ['XDG_RUNTIME_DIR'], "vsm-ipc.socket")
+IPC_SOCKET = "ipc://{}".format(IPC_SOCKET_PATH)
+
+def _send(signal, value):
+    context = zmq.Context()
+    socket = context.socket(zmq.REQ)
```

Comment:
As we discussed on IRC, this is not such high priority that we should spend too much time on it, so if it is working let's just keep it that way.